README.md

```diff
@@ -51,9 +51,12 @@ Generators:
 
 Once installation is complete, begin your journey!
 ```sh
-$ npm start
+$ node server.js
 ```
 
+`npm start` can also be used to start your application, but please note that it
```

Comment:
It has been added here https://github.com/trailsjs/trails-docs/blob/master/introduction/getting-started.md#third-step--run by @maissani :)